ares/GNUmakefile

```diff
@@ -1,7 +1,5 @@
 ifeq ($(platform),windows)
   options += -luuid -lkernel32 -luser32 -lgdi32 -lcomctl32 -lcomdlg32 -lshell32
-  options += -Wl,-enable-auto-import
```

Comment:
These seem to be very old MinGW options that no longer serve a purpose. They've been untouched for at least 12 years.